### PR TITLE
Update billing-rates.md

### PR DESCRIPTION
**Pull request by alexandercort**
# What does this PR do?

Updates mention that spot instances are only available for Runs. 

Adds a 'the' and a few '.'.

### CODE REVIEW

File: docs/platform/billing/billing-rates.md [hunk 1]
```diff
@@ -12,24 +12,20 @@ This page describes community edition pricing; refer to plans [here](https://www
...
 
-If your jobs can be interrupted and you want to save $$$$ then enable spot. Whenever the cloud provider needs that machine, the job will be killed automatically
+If your jobs can be interrupted and you want to save money then enable spot. Whenever the cloud provider needs that machine, the job will be killed automatically.
 
 ```text
 grid run --use_spot main.py
 ```
 
-:::note
-Interruptible machines are only available for Runs at the moment \(Sessions coming soon\)
```
Comment (oojo12) [inline docs/platform/billing/billing-rates.md:30]: Think it is fine to remove this. However, interruptible machines (spot instances) are available for Sessions now.

**Review by oojo12 — COMMENTED** [review]
leaving to @essiequoi to approve.

**Comment by essiequoi** [comment]
spot is available for sessions @alexandercort

**Comment by alexandercort** [comment]
@essiequoi I know and given that this statement is no longer valid.

**Review by essiequoi — APPROVED** [review]

**Pull request merged**